src/pykeen/models/unimodal/inductive_nodepiece_gnn.py

```diff
@@ -45,21 +45,26 @@ def __init__(
         """
         super().__init__(**kwargs)
 
-        train_factory, inference_factory, validation_factory, test_factory = kwargs.get('triples_factory', None), \
-                                                                             kwargs.get('inference_factory', None), \
-                                                                             kwargs.get('validation_factory', None), \
-                                                                             kwargs.get('test_factory', None)
+        train_factory, inference_factory, validation_factory, test_factory = (
+            kwargs.get("triples_factory", None),
```

Comment:
get actually gives None by default, so you might be able to cut this down a bit. Also consider using `pop(<key>, None` if you want to remove it from `kwargs` at the same time (note it does require the explicit default)

Reply:
https://github.com/pykeen/pykeen/pull/722/commits/84ed286844f748d8beed9c7eb40a9da43d3e2ec2